app.py

```diff
@@ -16,6 +19,12 @@ def index():
 def input():
     return render_template("input.html")
 
+@app.route("/sscrapy")
```

Comment:
Remove this one

Reply:
sir, this folder sscrapy consist of all the functions and it is routed in the app.py.
The oder folder scraping was deleted!
u can check the screenshot of what i'm talking about.

![Screenshot from 2023-11-22 10-43-09](https://github.com/ADORSYS-GIS/report-sscrap/assets/111261569/c7a6bb73-6e6d-41c0-a7ce-2f8ae44e5e83)